bundle/manifests/kuadrant-operator.clusterserviceversion.yaml

```diff
@@ -442,18 +442,10 @@ spec:
                     memory: 200Mi
                 securityContext:
                   allowPrivilegeEscalation: false
-                volumeMounts:
```

Comment:
so, latest controller-runtime and apimachinery remove the config file and settings will be passed command line?

Reply:
Yes, its a change from controller runtime where they have deprecated this, although we can use our own custom implementation instead
- https://github.com/Kuadrant/kuadrant-operator/pull/246#discussion_r1311594437